t/apicast-policy-3scale-batcher.t

```diff
@@ -0,0 +1,373 @@
+use lib 't';
+use Test::APIcast 'no_plan';
```

Comment:
Might be worth leaving a comment why we are not using blackbox here.

Reply:
I needed to set up the dictionaries with `lua_shared_dict` and also define some custom locations like `/force_report_to_backend` and `/check_batched_reports`. I think that defining those things is not possible using Blackbox tests.